src/components/common/ColonyActions/usePaginatedActions.ts

```diff
@@ -45,19 +50,23 @@ export const usePaginatedActions = (): UsePaginatedActionsReturn => {
     setSortDirection(newSortDirection);
   };
 
-  const loadMoreActions = () => {
+  if (hasMoreActions && filteredActions.length < actionsListSize) {
```

Comment:
If we don't have any more actions to fetch: no problem. Just display what we have.

If we do have more actions to fetch, and we don't yet have enough actions to display, then fetch some more actions. Keep going until we have enough to display to the user.